Assets/MRTK/Core/Utilities/Gltf/Schema/GltfAccessor.cs

```diff
@@ -100,5 +105,22 @@ public class GltfAccessor : GltfChildOfRootProperty
         /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/bufferView.schema.json
         /// </summary>
         public GltfBufferView BufferView { get; internal set; }
+
+        public void OnAfterDeserialize()
```

Comment:
Also, nit, but since these aren't methods we expect to be called manually, could these be [implemented explicitly](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation)? That way the methods are hidden from our docs and intellisense, but still callable by Unity.